### PR TITLE
Add DeletedSubscription resource model

### PR DESCRIPTION
**Pull request by tylercreller**

### CODE REVIEW

**Comment by rawsyntax** [comment]
@tbrisker @tzvatot Please take a look

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Changes
 
 This document describes the relevant changes between releases of the API model.
+## 0.0.241 Dec 08 2022
+- Add `AddonCluster` resource model.
+- Add `AddonClusters` locator in root resource model.
+- Add `Addon` locator in `AddonStatuses` resource model
+- Updated `AddOnStatusType` for `StatusConditions` and `AddonId`
```
[the remark this answers is not in the document]
Comment (tylercreller) [reply CHANGES.md:8]: Looks like there may have been additional changes on that PR, I can get those added in here as well

**Comment by Lir10** [comment]
lgtm

**Comment by tzvatot** [comment]
I'm missing a GET for DeletedSubscription. Every object in the list should have an HREF that is a target for a GET.

**Comment by tylercreller** [comment]
> I'm missing a GET for DeletedSubscription. Every object in the list should have an HREF that is a target for a GET.

DeletedSubscriptions from AMS does not support fetching single resources by id, so we only need List at the moment. Let me know if I am misunderstanding the ask.

**Comment by tzvatot** [comment]
> 

I

> > I'm missing a GET for DeletedSubscription. Every object in the list should have an HREF that is a target for a GET.
> 
> DeletedSubscriptions from AMS does not support fetching single resources by id, so we only need List at the moment. Let me know if I am misunderstanding the ask.

I suggest the AMS backend will support fetching a single resource by ID.

**Comment by tylercreller** [comment]
Created SDB-3364 as a future enhancement

**Comment by rawsyntax** [comment]
+1 to addressing it in a future PR

**Comment by cristianoveiga** [comment]
LGTM

**Pull request merged**